draft-barnes-acme.md

```diff
@@ -245,12 +245,13 @@ ACME is structured as a REST application with a few types of resources:
 * A "new-registration" resource
 * A "new-authorization" resource
 * A "new-certificate" resource
+* A "revoke-certificate" resource
 
 In general, the intent is for authorization and certificate resources to contain only public information, so that CAs may publish these resources to document what certificates have been issued and how they were authorized.  Non-public information, such as
 contact information, is stored in registration resources.
 
-In order to accomplish ACME transactions, a client needs to have the server's new-registration, new-authorization, and new-certificate URIs; the remaining URIs are provided to the client as a result of requests to these URIs.  To simplify
-configuration, ACME uses the "next" link relation to indicate URI to contact for the next step in processing: From registration to authorization, and from authorization to certificate issuance.  In this way, a client need only be configured with the registration URI.
+In order to accomplish ACME transactions, a client needs to have the server's new-registration, new-authorization, new-certificate, and revoke-certificate URIs; the remaining URIs are provided to the client as a result of requests to these URIs.  To simplify
+configuration, ACME provides a directory object. The client can fetch the directory object at startup to learn the URL for each resource.
```

Comment:
Please just rewrite this paragraph to say that ACME uses different URIs for different management functions; these functions are listed in a directory; and so clients only need to be configured with the directory URI.